lib/index.js

```diff
@@ -294,6 +296,19 @@ function _checkMaxLength(title, namespace) {
     }
 }
 
+function _normalizeSpecialPage(title, siteInfo) {
+    var parts = title.split('/');
+    var first = parts[0];
+    var alias = arrayFind(siteInfo.specialpagealiases, function(o) {
```

Comment:
This makes it a breaking change for old clients - they wouldn't provide the `specialpagealiases` key in the siteInfo and normalization will fail for special pages. So we need to either do a major version bump or add a check that `spacialpagealiases` and present indeed.